rules/consistent-test-it.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const { getNodeName, isTestCase, isDescribe } = require('./util');
+
+module.exports = {
+  meta: {
+    docs: {
+      url:
+        'https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/consistent-test-it.md',
+    },
+    fixable: 'code',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          fn: {
+            enum: ['it', 'test'],
+          },
+          withinDescribe: {
+            enum: ['it', 'test'],
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+  },
+  create(context) {
+    const configObj = context.options[0];
+    const testKeyword = (configObj && configObj.fn) || 'test';
+    const testKeywordWithinDescribe =
+      (configObj && configObj.withinDescribe) ||
+      (configObj && configObj.fn) ||
+      'it';
+
+    let describeNestingLevel = 0;
+
+    return {
+      CallExpression(node) {
+        const nodeName = getNodeName(node.callee);
+
+        if (isDescribe(node)) {
+          describeNestingLevel++;
+        }
+
+        if (
+          isTestCase(node) &&
+          describeNestingLevel === 0 &&
+          nodeName.indexOf(testKeyword) === -1
+        ) {
+          const opositeTestKeyword = getOpositeTestKeyword(testKeyword);
```

Comment:
`oposite` is misspelled - should be `opposite`.

Reply:
🙃